research-software-directory/RSD-as-a-service
# feat: add login options dialog

Changes proposed in this pull request:

* Adds login modal with different options to choose from.      

How to test:

* The behavior from surfconext should stay identical to the previous version. When login in from any page, after logging in the front end should return where it was. 

![image](https://user-images.githubusercontent.com/4195550/164480929-9c556548-18f7-4e26-8328-a971bbea2805.png)

PR Checklist:
